src/intervalToDuration/index.ts

```diff
@@ -33,49 +33,36 @@ import sub from '../sub/index'
  * })
  * // => { years: 39, months: 2, days: 20, hours: 7, minutes: 5, seconds: 0 }
  */
-
-export default function intervalToDuration({ start, end }: Interval): Duration {
+export default function intervalToDuration(interval: Interval): Duration {
   requiredArgs(1, arguments)
 
-  const dateLeft = toDate(start)
-  const dateRight = toDate(end)
+  const start = toDate(interval.start)
+  const end = toDate(interval.end)
```

Comment:
if you really want to keep the internal variable names the same (while keeping the func signature the same as before), you could do something like this:
```suggestion
  start = toDate(start)
  end = toDate(end)
```

Reply:
Personal preference, but I don't love assignments to function arguments. I tend to treat them as read only. In this case, they are also not quite the same type. Interval start and end are `Date | number` but the return value of `toDate` is `Date`.